depthai_nodes/ml/messages/classification.py

```diff
@@ -72,3 +72,21 @@ def scores(self, value: NDArray[np.float32]):
         if value.size > 0 and value.dtype != np.float32:
             raise ValueError("Scores must be a np.ndarray of floats.")
         self._scores = value
+
+    @property
+    def top_class(self) -> str:
+        """Returns the most probable class.
+
+        @return: The top class.
+        @rtype: str
+        """
+        return self._classes[0]
```

Comment:
Is the first class always the most probably one? Where do we define this?

Reply:
It is sorted in creator function [here](https://github.com/luxonis/depthai-nodes/blob/ef25b049badd86e0d92a679fb0bad2bb4ce65d7e/depthai_nodes/ml/messages/creators/classification.py#L80).

Reply:
I see, even though we normally always make the message by using the creator function, I think it's good that message also functions standalone. Thus, I think it's good we add a check to the Scores setter if the values are ordered from the biggest to smallest. Also, I'd write this order requirement in the Classifications message docstring.

Reply:
The `Classification` msg is also used in ClassificationSequenceParser and `create_classificaiton_sequence_msg` used for text recognition. So, the classes are not ordered by scores there. In this case, using `top_class` wont make sense. Maybe we update the `top_class` docstring with something like `Only works if scores are sorted by descending order`? I could also rename it to `first_class` but IMO `top_class` is better

Reply:
Hm true, I'd then go with the docstring explanation as calling the `top_class` method from the `ClassificationSequenceParser` doesn't really make sense (and thus users probably won't make this mistake too often).